beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -434,12 +434,15 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
             // target when there is no sampling peers available. This is a valid state and should not
             // return an error.
             return Ok(KeepChain);
-        } else {
-            return Err(RemoveChain::WrongChainState(format!(
-                "Batch not found for current processing target {}",
-                self.processing_target
```

Comment:
this will be a rare event, should we add a debug log? If we ever wonder why the SyncingChain is not processing anything

Reply:
Yeah good call.

Reply:
I don't think it will ever stop processing, something will progress it, but agree a log would be handy